libs/util/shared/src/lib/services/theme.service.ts

```diff
@@ -81,6 +82,14 @@ export class ThemeService {
         titleFont
       )
     }
+
+    if (fontsStylesheetUrl) {
```

Comment:
what if we need several fonts ?

Reply:
so if it's several google fonts you can specify them all in the same url; I don't know of any other apis like that but if the need arises then we can support an array of values; on the other hand, I'd be more keen in allowing custom fonts as files in the assets instead